### DISCUSSION TITLE
Error if launching Code while offline

### DISCUSSION THREAD

**Discussion by tricki** [community/community#14545] [Copilot Conversations]
I started VS Code while I was offline, so Copilot couldn't connect. But even after I was connected the only way of getting Copilot to work was restarting VS Code. I would have expected something like a "Try reconnecting" button.

The error message was: `GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo ENOTFOUND api.github.com"`

Copilot Labs worked fine without a restart.

**Comment by k-loki** [comment]
yeah, I was suffering the same... 
would love something like 'reconnect' or some other way without restarting vscode.

**Reply by foxt** [reply]
You can `Restart Extension Host` from the `F1` menu instead of restarting the entire editor, but its still a hassle.
